### PR TITLE
Change yamllint image details

### PR DESCRIPTION
**Pull request by Rajalakshmi-Girish**
This change contains:
- Dockerfile and README for yamllint image at `quay.io/powercloud/yamllint:1.34.0`
- Correct image name for `pull-test-infra-yamllint` presubmit job
- Correct the manifests `deck_ingress.yaml` and `hook_ingress.yaml` to avoid error from yamllint tool `wrong new line character: expected \n  (new-lines)`

### CODE REVIEW

File: images/yamllint/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM python:3.13.0a3-bookworm
```
Comment (mkumatag) [inline images/yamllint/Dockerfile:1]:
```suggestion
FROM python:3-bookworm
```

**Review by mkumatag — CHANGES_REQUESTED** [review]

File: images/yamllint/Dockerfile (hunk 1 above)
Comment (Rajalakshmi-Girish) [reply images/yamllint/Dockerfile]: @mkumatag I have changed and pushed a new image.

**Review by mkumatag — APPROVED** [review]
/lgtm

**Comment by mkumatag** [comment]
@Rajalakshmi-Girish not sure why PR is not getting merged!

**Comment by Rajalakshmi-Girish** [comment]
> @Rajalakshmi-Girish not sure why PR is not getting merged!

Not sure why labels are not added too. Will check.

**Pull request merged**